core/src/main/scala/fs2/job/JobManager.scala

```diff
@@ -134,11 +141,12 @@ final class JobManager[F[_]: Concurrent: Timer, I, N] private (
     Concurrent[F].delay(Option(meta.get(id)).map(_.status))
 
   private def shutdown: F[Unit] =
-    Concurrent[F].delay(meta.clear()) >> notificationsQ.enqueue1(None)
+    Concurrent[F].delay(meta.clear()) >> notificationsQ.enqueue1(None) >> eventQ.enqueue1(None)
```

Comment:
Random thought: this will mean that `shutdown` cannot run if the queues are blocked. We may want to consider `notificationsQ.enqueue1(None).start` (and analogously for `eventQ`) just to make sure that we can always instantly shut down a job manager.

Reply:
`.start` from [`Concurrent#start`](https://github.com/typelevel/cats-effect/blob/master/core/shared/src/main/scala/cats/effect/Concurrent.scala#L208)? Wouldn't it still block if one of the queues is full? Or should we call cancel on the returned fiber after some timeout?

Reply:
The spawned fiber would block if one of the queues is full, but that's okay. It's not like we're going to call `shutdown` in a loop. This at least guarantees that we can always stop the job manager without blocking.

Reply:
I see. I'll make this change.

Reply:
This is done. Including tests verifying we can  `shutdown` a `JobManager` when event/notification queues are full.